shoop/admin/modules/shops/views/list.py

```diff
@@ -25,3 +27,6 @@ class ShopListView(PicotableListView):
         Column("identifier", _(u"Identifier")),
         Column("status", _(u"Status"), filter_config=ChoicesFilter(choices=ShopStatus.choices)),
     ]
+
+    def get_toolbar(self):
+        return super(ShopListView, self).get_toolbar() if settings.SHOOP_ENABLE_MULTIPLE_SHOPS else Toolbar([])
```

Comment:
Use real if rather than ternary-operator for this, i.e.:
```python
if settings.SHOOP_ENABLE_MULTIPLE_SHOPS:
    super(ShopListView, self).get_toolbar()
else:
    return Toolbar([])
```
because IMHO that is easier to read.